### ISSUE TITLE
Increasing color contrast of the Toast component in dark and high contrast mode

### ISSUE THREAD

**Issue by runningonsideprojects** [processing/p5.js-web-editor#3857]
### Increasing Access

This feature enhancement would increase accessibility to visually-impaired users by increasing the color contrast to adhere to WCAG guidelines. 

### Issue Description

This specific subissue aims to address the color contrast issues with the Toast component. WCAG guidelines [state](https://www.allaccessible.org/blog/color-contrast-accessibility-wcag-guide-2025) that non-text color contrast should be at least 3:1. 

In dark mode, the toast background color (`$medium-dark`), with the page background color (`$dark`) has a color contrast of 1.49:1 and does not meet the criteria of 3:1. In high contrast mode, the toast background color (`$medium-dark`), with the page background color (`$darker`) has a color contrast of 2.01:1 and does not meet the criteria of 3:1. 

#### Dark mode: 
- [ ] Change the toast background color from `$middle-dark` to `$middle-light`
- [ ] Change toast text color from `$lightest` to `$dark`

This should enable a color contrast of 5.19:1. 

<img width="512" height="51" alt="Image" src="https://github.com/user-attachments/assets/3e4f5c63-1b54-4f68-8226-39dacfef7758" />

#### High contrast mode: 
- [ ] Change the toast background color from `$middle-dark` to `$middle-light`
- [ ] Change toast text color from `$lightest` to `$darker`

This should enable a color contrast of 7:1. 

<img width="512" height="59" alt="Image" src="https://github.com/user-attachments/assets/041adced-8931-42f9-af67-e470c5667bb3" />

### Files to change
- `client/styles/abstracts/_variables.scss`, changing `$toast-background-color` and `$toast-text-color`

### PR Criteria
- [ ] Provide screenshots in ur PR of the visual changes implemented
- [ ] Provide QA steps for how reviewers can ensure your changes are made properly

### Additional Resources:
- [Color Combinations for Accessibility](https://github.com/processing/p5.js-web-editor/wiki/Color-Combinations-for-Accessibility) – tool developed by @Izzy-Snyder that generates custom color palettes for components while adhering to WCAG guidelines and [p5.js](http://p5.js/) styling standardizations. 
- [Zeplin Style Link](https://scene.zeplin.io/project/55f746c54a02e1e50e0632c3) – the p5.js editor’s styling standardizations
- [Preparing a Pull Request](https://github.com/processing/p5.js-web-editor/blob/develop/contributor_docs/preparing_a_pull_request.md) – how-to guide on creating a Pull Request 
- [Style Guide Documentation](https://github.com/processing/p5.js-web-editor/blob/develop/contributor_docs/styling.md) (WIP), best-practices guide on implementing styling changes to the codebase.

**Comment by runningonsideprojects** [comment]
@raclim hey Rachel, i can take a stab at this one if thats ok!

**Comment by raclim** [comment]
@runningonsideprojects yes please go ahead! :)
